src/theme/stories/spacingShowcase.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from 'react';
+import { Box, Td, Tr, useTheme } from '@chakra-ui/react';
+
+type Props = {
+  name: string;
+};
+
+const SpacingShowCase: FC<Props> = ({ name }) => {
+  const theme = useTheme();
+  const unitRem = parseFloat(theme.spacing[name]);
+  const unitPixel = unitRem / 0.0625;
+  return (
+    <Tr border="1px" borderColor={theme.colors.gray[100]}>
+      <Td>{name}</Td>
+      <Td>{`${unitRem}rem`}</Td>
+      <Td>{`${unitPixel}px`}</Td>
```

Comment:
This is very confusing to me. `rem` has the advantage that it considers the user's font size preference. So this value is correct if you have "medium" but wrong if I increase my font size to large or extra-large. Maybe you can use `getComputedStyle` to get the correct px size?

Reply:
As we discussed,  I will add a text where I explain that this is ONLy a reference table, then we won't have more confusions. Thank you for the idea/suggestion